src/applications/personalization/dashboard/components/IconCTALink.jsx

```diff
@@ -36,21 +36,19 @@ const IconCTALink = ({
       data-testid={testId || ''}
     >
       <span className="vads-u-display--flex">
-        <span className="fa-stack fa-sm vads-u-height--full vads-u-margin-right--1">
-          <i
-            aria-hidden="true"
-            className="fas fa-circle fa-stack-2x vads-u-color--primary-alt-lightest"
-          />
-          <i aria-hidden="true" className={`fas fa-${icon} fa-stack-1x`} />
+        <span className="vads-u-margin-x--1 vads-u-margin-top--0p5">
+          <va-icon icon={`${icon}`} size={2} srtext={`${icon}`} />
```

Comment:
is there a reason for using the template literal here?

```
<va-icon icon={`${icon}`} size={2} srtext={`${icon}`} />
```
vs
```
<va-icon icon={icon} size={2} srtext={icon} />
```

Otherwise everything else looks good!